src/components/preview/preview.tsx

```diff
@@ -48,26 +87,28 @@ const Preview = ({
   width,
   height,
   scale,
-  pageNumber,
+  page,
   selectedPage,
   previewPage,
   setSelectedPage,
   onPanelPropertiesChange,
-  deletePage
+  deletePage,
+  addQuestion
 }: Props) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement>();
   const [pageMenuOpen, setPageMenuOpen] = useState(false);
+  const [addQuestionDialogOpen, setAddQuestionDialogOpen] = useState(false);
 
   /**
    * Set up event listener to recieve post message from iframe
    */
   useEffect(() => {
     // TODO: In the future when we need more event handlers within the iframe we can use a switch statement to replace the handlePostMessageEventListener.
     // This can direct us to the appropriate event listener based upon the type of the Event e.g. IframeClickEvent, IframeButtonClickEvent etc.
-    window.addEventListener(`message-${pageNumber}`, handlePostMessageEventListener);
+    window.addEventListener(`message-${page?.orderNumber}`, handlePostMessageEventListener);
 
     return () =>
-      window.removeEventListener(`message-${pageNumber}`, handlePostMessageEventListener);
+      window.removeEventListener(`message-${page?.orderNumber}`, handlePostMessageEventListener);
```

Comment:
There are going to be merge conflicts between this branch and feature 25 in this file